crates/boundless-cli/tests/povw.rs

```diff
@@ -308,8 +307,161 @@ async fn claim_reward_multi_epoch() -> anyhow::Result<()> {
         final_balance > alloy::primitives::U256::ZERO,
         "Value recipient should have received tokens"
     );
-    println!("✓ Multi-epoch claim-reward test completed. Final balance: {}", final_balance);
+    println!("✓ Multi-epoch claim test completed. Final balance: {}", final_balance);
+
+    Ok(())
+}
+
+/// Test that if mint is called when one update is in a finalized epoch, and a second update is in
+/// an unfinalized epoch, that the process succeeds but provides a warning about the skipped epoch.
+#[tokio::test]
+async fn claim_on_partially_finalized_epochs() -> anyhow::Result<()> {
```

Comment:
Tests in this file are pretty boiler-plate heavy right now. I generally find this way of doing CLI testing very helpful, but it needs some cleanup to make it easy to keep adding tests.